tests/app/test_model.py

```diff
@@ -65,3 +71,96 @@ def test_status_conversion_handles_failed_statuses(initial_statuses, expected_st
     converted_statuses = Notification.substitute_status(initial_statuses)
     assert len(converted_statuses) == len(expected_statuses)
     assert set(converted_statuses) == set(expected_statuses)
+
+
+@freeze_time("2016-01-01 11:09:00.000000")
+@pytest.mark.parametrize('template_type, recipient', [
+    ('sms', '+447700900855'),
+    ('email', 'foo@bar.com'),
+])
+def test_notification_for_csv_returns_correct_type(notify_db, notify_db_session, template_type, recipient):
+    template = create_sample_template(notify_db, notify_db_session, template_type=template_type)
+    notification = create_sample_notification_with_job(
+        notify_db,
+        notify_db_session,
+        template=template,
+        to_field=recipient
+    )
+
+    expected_resp = {
+        "recipient": recipient,
+        "row_number": "",
+        "template_name": "Template Name",
+        "template_type": template_type,
+        "created_at": "Friday 01 January 2016 at 11:09",
+        "job_name": "some.csv",
+        "status": "Sending"
+    }
+
+    serialized = notification.serialize_for_csv()
+    assert serialized == expected_resp
```

Comment:
Perhaps just do an assertion on the template_type in case underlying sample changes.